src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsFromServer(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
```

Comment:
no needs to return `null` since if you don't find user you needed you'll get falsy value anyway
```suggestion
  return usersFromServer.find(user => user.id === userId);
}

function getCommentsFromServer(postId) {
  return (
    commentsFromServer.filter(comment => comment.postId === postId)
```